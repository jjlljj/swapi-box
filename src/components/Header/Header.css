.site-header {
  padding: 10px 0 0;
  text-align: center;
}

.banner-text {
  font-family: 'Pathway Gothic One', sans-serif;
  font-size: 62px;
  letter-spacing: 4px;
}

.banner-text a {
  
}

.favorites-nav-btn {
  background: black;
  margin: 20px;
}

.favorites-nav-btn a {
  display: block;
  padding: 6px;
  width: 100px;
  height:100%;
}