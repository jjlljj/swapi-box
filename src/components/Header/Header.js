import React from 'react';
import './Header.css';
import Nav from '../Nav/Nav';
import { Link, NavLink } from 'react-router-dom';

const Header = () => {
  return (
    <div>
      <header className="site-header">
        <h1><Link to='/'>STAR WARS</Link></h1>
        <div className="favorites-nav-btn">
          <NavLink 
            to="/favorites"
            activeClassName="active">
            Favorites
          </NavLink>
        </div>
      </header>    
      <Nav />
    </div>
  );
};

export default Header;
