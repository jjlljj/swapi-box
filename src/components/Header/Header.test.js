/* eslint-disable */

import React from 'react';
import ReactDom from 'react-dom';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {

  it('should exist', () => {
    const renderedComponent = shallow(<Header />);

    expect(renderedComponent).toBeDefined();
  })

})
