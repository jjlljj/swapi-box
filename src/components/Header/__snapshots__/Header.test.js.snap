// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          replace={false}
          to="/"
>
          <img
                    alt="Star wars"
                    className="banner-text"
                    src="star-wars-logo-red.png"
          />
</Link>,
        <Nav
          favLength={undefined}
/>,
      ],
      "className": "site-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <img
            alt="Star wars"
            className="banner-text"
            src="star-wars-logo-red.png"
/>,
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Star wars",
            "className": "banner-text",
            "src": "star-wars-logo-red.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "favLength": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            replace={false}
            to="/"
>
            <img
                        alt="Star wars"
                        className="banner-text"
                        src="star-wars-logo-red.png"
            />
</Link>,
          <Nav
            favLength={undefined}
/>,
        ],
        "className": "site-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <img
              alt="Star wars"
              className="banner-text"
              src="star-wars-logo-red.png"
/>,
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Star wars",
              "className": "banner-text",
              "src": "star-wars-logo-red.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "favLength": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
